Cargo.toml

```diff
@@ -15,6 +15,6 @@ default = []
 web = []
 
 [dependencies]
-epi = "0.15"
+epi = "0.16"
```

Comment:
nit, but `epi` is no longer used with the removal of the `TextureAllocator` trait implementation. It's re-exported, and uses the `egui` export, that's it.

The one downside to the removal of that impl is that the demo app will no longer work. See: https://github.com/hasenbanck/egui_example/blob/78e4e27a7733fcef14c6c937527ba74429e7aa0c/src/main.rs#L135

This isn't an issue for me personally, but I know some people do expect to use `epi`.

Reply:
I think we should then directly use the egui crate instead. This will be a new major release, so breaking things is fine. People can just pull epi in their projects instead if they really need it.